tests/unit/components/g-map-infowindow-test.js

```diff
@@ -168,6 +175,34 @@ test('it triggers `setMarker` on `mapContext.marker` change', function() {
   sinon.assert.calledOnce(component.setMarker);
 });
 
+test('it triggers `setOptions` on `disableAutoPan` change', function() {
+  component.setOptions = sinon.spy();
+  run(() => component.set('disableAutoPan', true));
+  sinon.assert.calledOnce(component.setOptions);
+});
+
+test('it triggers `setOptions` on `maxWidth` change', function() {
+  component.setOptions = sinon.spy();
+  run(() => component.set('maxWidth', 200));
+  sinon.assert.calledOnce(component.setOptions);
+});
+
+test('it triggers `setOptions` on `pixelOffset` change', function() {
+  component.setOptions = sinon.spy();
+  run(() => component.set('pixelOffset', { width: 10, height: 10 }));
+  sinon.assert.calledOnce(component.setOptions);
+});
+
+test('it doesn\'t call `setOptions` of InfoWindow on `setOptions` when no option present', function() {
```

Comment:
Could you also add test case for when it actually calls `setOptions` of InfoWindow with provided options?